File: abc333/src/a.cpp
#include <iostream>
#include <string>

int main()
{
    int n;
    std::cin >> n;

    if (n < 1 || n > 9)
    {
        return 1;
    }

    std::string result;
    for (int i = 0; i < n; ++i)
    {
        result += std::to_string(n);
    }

    std::cout << result << std::endl;

    return 0;
}

File: abc333/src/bin/a.rs
use std::io;

fn main() {
    let mut input = String::new();
    io::stdin()
        .read_line(&mut input)
        .expect("Failed to read input");
    let n: i32 = input.trim().parse().expect("Invalid input");

    if n < 1 || n > 9 {
        return;
    }

    let result: String = std::iter::repeat(n.to_string()).take(n as usize).collect();

    println!("{}", result);
}
